scipy/stats/tests/test_binned_statistic.py

```diff
@@ -44,7 +44,7 @@ def test_big_number_std(self):
         # see issue gh-10126 for more
         x = self.x
         u = self.u
-
+        print(u)
```

Comment:
```suggestion
```

Reply:
Ooops left one in. I was getting local test failures in that module.